ci-operator/jobs/openshift/cluster-api-provider-baremetal/openshift-cluster-api-provider-baremetal-main-presubmits.yaml

```diff
@@ -669,11 +669,11 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )okd-scos-e2e-aws-ovn,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Hmm, why is this done? Looks unrelated to me.

Reply:
this should be overridden by `skip_if_only_changed`, so not taken into consideration
I agree the change is confusing

Reply:
that was done by the `make jobs` step, but I'll put it back. Hopefully, CI is ok with it.